src/EA.Iws.Web/Web.config

```diff
@@ -15,7 +15,7 @@
   </configSections>
   <system.web>
     <compilation debug="true" targetFramework="4.5.2" />
-    <httpRuntime targetFramework="4.5.2" enableVersionHeader="false" encoderType="System.Web.Security.AntiXss.AntiXssEncoder" maxRequestLength="51200" />
+    <httpRuntime targetFramework="4.5.2" enableVersionHeader="false" encoderType="System.Web.Security.AntiXss.AntiXssEncoder" maxRequestLength="2147483647" executionTimeout="600" />
```

Comment:
@GonzaloGirasol - Worth being aware of the execution timeout property here. Set to 10mins (value is in seconds). Any concerns?

Reply:
Probably set this to 5 minutes. I feel like 10 is too long. But then again I cant' really say at this point, it's hard to judge how well it's going to perform...

Reply:
Actually on this note, just leave it as it is, we've got no idea at this stage..